README.md

```diff
@@ -87,7 +120,7 @@ fields to get the information they want.
 Unit Tests
 ----------
 
-You can run the tests via the setup.py script:
+It's a good idea to write and run unit tests on the ducktape framework itself. You can run the tests via the setup.py script:
```

Comment:
Yes it is :) We might want to split the readme into user and developer sections since right now they're kind of mixed together.